src/analysis/thermo/vonBertalanffy/initVonBertalanffy.m

```diff
@@ -195,3 +58,12 @@
     fprintf('%s\n','The solution will be arch dependent');
 end
 
+[status,result] = system('babel');
```

Comment:
Wouldn't it make sense to put this right at the start (because if the requisits are not matched, I would avoid changing pathes etc.

Reply:
Ok, I will take a look

Reply:
So the only thing that in my opinion makes sense would be to put the first part in the end because all the others are similar - checking for requisites.